src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -147,31 +153,73 @@ class StencilTest extends GPUTest {
     };
 
     const encoder = this.device.createCommandEncoder();
-    const pass = encoder.beginRenderPass({
-      colorAttachments: [
-        {
-          view: renderTarget.createView(),
-          storeOp: 'store',
-          loadOp: 'load',
+    let pass: GPURenderPassEncoder;
+    if (isSingleEncoderMultiplePass) {
+      // Clear the depthTextureView at the beginning.
```

Comment:
Is this necessary? It should be cleared to 0 when it's created.

Reply:
Ah, then we don't need to do this. I referred to the end2end test.